plugins/webp-uploads/hooks.php

```diff
@@ -533,29 +533,28 @@ function webp_uploads_update_image_references( $content ): string {
 	}
 
 	// This content does not have any tag on it, move forward.
+	// TODO: Eventually this should use the HTML API to parse out the image tags and then update them.
```

Comment:
or, switch to using the `wp_content_img_tag` filter? https://github.com/WordPress/performance/issues/1259